src/components/SlateEditor/plugins/detailsbox/DetailsBox.jsx

```diff
@@ -6,26 +6,161 @@
   };
 
+  const summary = node.findDescendant(node => {
+    if (node.type === 'summary') return node;
+  });
+
+  const [open, setOpen] = useState(true);
+  const [showEditModal, setShowEditModal] = useState(false);
+
+  const toggleOpen = () => {
+    setOpen(!open);
+  };
+
+  const summaryTextNode = summary.getLastText();
+  const [inputValue, setInputvalue] = useState(summary.text);
+
+  const onChangeSummary = () => {
+    const newTextNode = Text.create({
+      text: inputValue,
+    });
+    editor.replaceNodeByKey(summaryTextNode.key, newTextNode);
+    setShowEditModal(false);
+  };
+
+  const [summaryNode, ...contentNodes] = props.children;
+
   return (
-    <details {...props.attributes} open>
-      {props.children}
+    <div css={detailsWrapper} {...props.attributes}>
+      <StyledRow>
+        <summary css={summaryStyle(open)} onClick={toggleOpen}>
+          {summaryNode}
+        </summary>
+        {open && (
+          <Button
+            css={css`
+              height: 100%;
+              display: none;
+            `}
+            onClick={() => setShowEditModal(true)}
+            stripped>
+            <Pencil />
+          </Button>
+        )}
+      </StyledRow>
+      <Portal isOpened>
+        <Modal controllable isOpen={showEditModal}>
+          {props => (
+            <>
+              <ModalHeader>
+                {' '}
+                <ModalCloseButton
+                  title={t('dialog.close')}
+                  onClick={() => setShowEditModal(false)}
+                />
+              </ModalHeader>
+              <ModalBody>
+                <Input
+                  name="caption"
+                  container="div"
+                  label="Endre overskrift"
+                  type="text"
+                  value={inputValue}
+                  onChange={e => setInputvalue(e.target.value)}
+                  placeholder={'Kort sammendrag'}
+                />
+                <StyledButtonWrapper paddingLeft>
+                  <Button onClick={() => setShowEditModal(false)} outline>
+                    {t('form.abort')}
+                  </Button>
+                  <Button onClick={onChangeSummary}>
+                    {t('form.image.save')}
+                  </Button>
+                </StyledButtonWrapper>
+              </ModalBody>
+            </>
+          )}
+        </Modal>
+      </Portal>
+      <div css={contentStyle(open)}>{contentNodes}</div>
       <DeleteButton stripped onMouseDown={onRemoveClick} />
-    </details>
+    </div>
   );
 };
+
+const detailsWrapper = css`
+  position: relative;
+  margin: ${spacing.large} 0;
+  padding-left: 26px;
+  min-height: 90px;
+  border: 1px solid ${colors.brand.greyLight};
+  overflow: hidden;
+
+  > *:last-child {
+    margin-bottom: 0;
+  }
+`;
+
+const contentStyle = open => css`
+  display: ${open ? '' : 'none'};
+  margin-top: calc(${spacing.small} * 1.5 + '0px');
+`;
+
+const summaryStyle = open => css`
+  color: #20588f;
```

Comment:
Hvilken farge er dette? Har vi variabler for den?

Reply:
Det er browser-fargen for summary-header :)